src/maven/maven-url.js

```diff
@@ -19,7 +19,24 @@ const createMavenUrlFromCoordinates = async coordinates => {
   }
 }
 
+const createMavenArtifactsUrlFromCoordinates = async coordinates => {
+  const pathifiedGroupId = coordinates.groupId?.replace(/\./g, "/")
+
+  const url = `https://repo1.maven.org/maven2/${pathifiedGroupId}/${coordinates.artifactId}/${coordinates.version}/`
```

Comment:
If I were you, I would get the `pom.xml` instead of the directly listing. I'm sure the files are cached in the CDN, I'm not sure about the directory listing.

Reply:
Ah, that's a good thought about the caching, thanks. I'd been trying to work out which would be most performant, and I decided the directory listing would be a smaller amount of data to send. But I never thought about caching. And, in fact, since it's a `HEAD` call the file size may not even matter.

Reply:
Done! https://github.com/quarkusio/extensions/pull/121